docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,506 @@
+      },
+      {
+        name: 'role',
+        type: `"button" | "link"`,
+        description: `Allows user interaction with the component. See the [role](#Role) variant to learn more.`,
+      },
+
+      {
+        name: 'accessibilityControls',
+        type: 'string',
+        description: `Specify the \`id\` of an associated element (or elements) whose contents or visibility are controlled by a component so that screen reader users can identify the relationship between elements. Optional with button-role component.
+
+See the [Accessibility guidelines](#Accessibility) for details on proper usage.`,
+      },
+      {
+        name: 'accessibilityExpanded',
+        type: 'boolean',
+        description: `Indicate that a component hides or exposes collapsible components and expose whether they are currently expanded or collapsed. Optional with button-role component.
+
+See the [Accessibility guidelines](#Accessibility) for details on proper usage.`,
+      },
+      {
+        name: 'accessibilityHaspopup',
+        type: 'boolean',
+        description: `Indicate that a component controls the appearance of interactive popup elements, such as menu or dialog. Optional with button-role component.
+
+See the [Accessibility guidelines](#Accessibility) for details on proper usage.`,
+      },
+      {
+        name: 'onClick',
+        type:
+          '({ event: SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> | SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>, {| disableOnNavigation: () => void |}> }) => void',
+        description: `Callback fired when the component is clicked (pressed and released) with a mouse or keyboard. See the [role](#Role) variant to learn more.`,
+      },
+      {
+        name: 'href',
+        type: 'string',
+        description: `When supplied, wraps the component in a link, and directs users to the url when item is selected. See the [role](#Role) variant to learn more.`,
+      },
+      {
+        name: 'addCollaborators',
+        type: 'boolean',
+        description: `When supplied, it appends an 'add' icon to the avatar pile as a call to action to the user. See [Best Practices](#Best-practices) for more info.`,
+      },
+      {
+        name: 'ref',
+        type: `React.Ref<'div'> | React.Ref<'a'>`,
+        description: `Forward the ref to the underlying div or anchor element. See the [role](#Role) variant to learn more.`,
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+Avatar Group requires \`accessibilityLabel\`. Avatar Group is a group of elements that require a parent label describing both the data presented and the call to action in the case of button and link roles. As seen in the example below, the screen-reader reads: "Group collaborators: Keerthi, Alberto, and 10 more. Add collaborators to this board."
```

Comment:
Nit here and throughout to remove "Group" from the accessibility labels, just "Collaborators: ..." is good

Reply:
DONE